scm/src/GFS_typedefs.F90

```diff
@@ -6893,7 +6911,7 @@ subroutine diag_phys_zero (Diag, Model, linit, iauwindow_center)
     Diag%dku = zero
 
 ! max hourly diagnostics
-    Diag%refl_10cm   = zero
+    Diag%refl_10cm   = -35.
```

Comment:
This change makes sense to me, but I want to check with @grantfirl if there was a particular reason to reset `Diag%refl_10cm` to zero instead of `-35.`

Reply:
As FYI to this point. Many people think dbz=zero means no precip. That is not true.  Very light drizzle occurs all the time with dbz of about -5 to -10 values.  The -35 value is considered (approximately) the lowest sensitivity level of the USA's NEXRAD radars.  Cloud radars are sensitive down to even smaller values, but the frequency/wavelength of NEXRAD is 10cm, hence why I was so specific in its name.

Reply:
No, @gthompsnWRF reasoning makes perfect sense. This was only zero in the SCM version of GFS_typedefs to match FV3.